core/templates/components/forms/custom-forms-directives/image-receiver.component.spec.ts

```diff
@@ -289,13 +302,13 @@ describe('ImageUploaderComponent', () => {
   });
 
   it('should not upload image if the size is more than 1MB for blog post', () => {
-    spyOn(contextService, 'getEntityType').and.returnValue('blog_post');
```

Comment:
Since there's no more special case for the blog post, maybe this test should be deleted?

Reply:
I guess we should keep it as for blog image the max size is 1Mb and all the other places we limit it to 100KB. so I believe 2 tests make sense.